index.html

```diff
@@ -11,10 +11,15 @@
 <h1>todos</h1>
 
 <div class="tasks">
-	<div class="task_new"><label> ˅ </label><em> What needs to be done?</em></div>
-	<div class="task_completed"><input type="checkbox" checked>to learn HTML</div>
+	<div class="task_new">
+		<label for="task_add_new"> ˅ </label>
+		<em><input id="task_add_new" placeholder="What needs to be done?" onkeypress="add_task()"></em>
```

Comment:
it's better to avoid using id, and use css instead of tag em

Reply:
changed